operator/src/main/java/oracle/kubernetes/operator/helpers/JobHelper.java

```diff
@@ -249,12 +249,8 @@ private String getAsServiceName() {
       addEnvVar(vars, IntrospectorJobEnvVars.NAMESPACE, getNamespace());
       addEnvVar(vars, IntrospectorJobEnvVars.INTROSPECT_HOME, getIntrospectHome());
       addEnvVar(vars, IntrospectorJobEnvVars.CREDENTIALS_SECRET_NAME, getWebLogicCredentialsSecretName());
-      addEnvVar(vars, IntrospectorJobEnvVars.OPSS_WALLET_SECRET_NAME, getOpssWalletSecretName());
-      addEnvVar(vars, IntrospectorJobEnvVars.WDT_ENCRYPTION_PASSPHRASE_NAME, getWdtEncryptPassPhraseName());
-      addEnvVar(vars, IntrospectorJobEnvVars.ROLLBACK_IF_REQUIRE_RESTART,
-          Boolean.toString(isRollBackIfRequireRestart()));
-      addEnvVar(vars, IntrospectorJobEnvVars.USE_ONLINE_UPDATE, Boolean.toString(isUseOnlineUpdate()));
-      addEnvVar(vars, IntrospectorJobEnvVars.KEEP_JRF_SCHEMA, Boolean.toString(isKeepJrfSchema()));
+      addEnvVar(vars, IntrospectorJobEnvVars.OPSS_KEY_SECRET_NAME, getOpssWalletSecretName());
```

Comment:
I think we agreed to change this to walletFileSecret and walletPasswordSecret per slack thread https://proddev-paas-fmw.slack.com/archives/CKFA73SNM/p1580504991015300, so we'll need to env vars OPSS_WALLET_FILE_SECRET_NAME and OPSS_WALLET_PASSWORD_SECRET_NAME.   Also Johnny's RCU code will eventially need to change to look in the right locations and use the right env vars to get these values.    Finally, the expected attribute within each secret will be 'walletFile' and 'walletPassword' respectively.